models/stg_hubspot__engagement_call.sql

```diff
@@ -14,31 +14,17 @@ with base as (
                 staging_columns=get_engagement_call_columns()
             )
         }}
+        {% if fivetran_utils.remove_prefix_from_columns(columns=adapter.get_columns_in_relation(ref('stg_hubspot__engagement_call_tmp')), prefix='property_hs_',exclude=get_macro_columns(get_engagement_call_columns())) | length > 0 %},{% endif %}
+        {{ 
+            fivetran_utils.remove_prefix_from_columns(
+                columns=adapter.get_columns_in_relation(ref('stg_hubspot__engagement_call_tmp')), 
+                prefix='property_hs_',exclude=get_macro_columns(get_engagement_call_columns()))
+        }}
```

Comment:
I just wanted to explain this a bit as there is a lot going on. Essentially what we are doing here is first checking if the results of the `fivetran_utils.remove_prefix_from_columns` macro produce any records. If not, then we don't need to worry about more fields being added. If they do, then we add a comma to allow for the fields to be selected properly.

The `fivetran_utils.remove_prefix_from_columns` macro simply will get all the columns from the relation (this being the tmp model) and will remove the `property_hs_` prefix from the fields (as almost all fields in the new api endpoint have this prefix). However, there may be some fields that we already called out in the `get_engagement_call_columns` macro that we don't want to bring in again. Therefore, the exclude argument is getting the results from that macro and excluding any fields that are the same.